### PR TITLE
Grammatical Errors Fixed

### PR DESCRIPTION
**Pull request by corbinjava**
Fixed all the English grammatical errors for GrimAC.

### CODE REVIEW

File: src/main/resources/config/en.yml [hunk 1]
```diff
@@ -37,31 +37,31 @@ spectators:
...
   # Measured in blocks from the possible movement
   # We account for Optifine by switching trig tables but dropping this to 0.001 will reduce FastMath
   # flagging the anticheat if this compensation doesn't work...
   threshold: 0.001
-  # How large of a violation in a tick before the player gets immediately setback?
+  # How large of a violation in a tick before the player gets immediately set back?
   # -1 to disable
   immediate-setback-threshold: 0.1
-  # How large of an advantage over all ticks before we start to setback?
+  # How large of an advantage over all ticks before we start to set back?
```
Comment (ManInMyVan) [inline src/main/resources/config/en.yml:56]: setback is a word

File: src/main/resources/messages/en.yml [hunk 2]
```diff
@@ -9,13 +9,13 @@ client-brand-format: "%prefix% &f%player% joined using %brand%"
 console-specify-target: "%prefix% &cYou must specify a target as the console!"
 player-not-found: "%prefix% &cPlayer is exempt or offline!"
 player-not-this-server: "%prefix% &cPlayer is not on this server!"
-spectate-return: "\n%prefix% &fClick here to return to previous location\n"
+spectate-return: "\n%prefix% &fClick here to return to the previous location\n"
```
Comment (ManInMyVan) [inline src/main/resources/messages/en.yml:12]: I think "your" would be better here

File: src/main/resources/config/en.yml [hunk 3]
```diff
@@ -155,23 +155,23 @@ Reach:
...
+# Enable logging plugins that have injected into netty on join to debug compatibility issues
 debug-pipeline-on-join: false
 
 # Enables experimental checks
 experimental-checks: false
 
-# Grim sometimes cancels illegal packets such as with timer, after X packets in a second cancelled, when should
-# we simply kick the player? This is required as some packet limiters don't count packets cancelled by grim.
+# Grim sometimes cancels illegal packets such as with timer, after X packets in a second canceled, when should
+# we simply kick the player? This is required as some packet limiters don't count packets canceled by Grim.
```
Comment (ManInMyVan) [inline src/main/resources/config/en.yml:174]: cancelled is fine, I see no need to change it

File: src/main/resources/config/en.yml [hunk 4]
```diff
@@ -37,31 +37,31 @@ spectators:
 max-transaction-time: 60
 
 # Should the duplicate movement packet be cancelled?
-# Mojang has fixed this issue in 1.21. This was their attempt to fix the "bucket desync". https://bugs.mojang.com/browse/MC-12363
+# Mojang has fixed this issue in 1.21. This was their attempt to fix the "bucket desync." https://bugs.mojang.com/browse/MC-12363
```
Comment (SamB440) [inline src/main/resources/config/en.yml:40]: This isn't correct, the full stop isn't a part of the quotation, but the sentence.

**Review by SamB440 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**